docs/sources/contribute/release-notes/index.md

```diff
@@ -34,185 +34,102 @@ The Grafana Cloud Traces updates are published using the What's new.
 
 ## What's new documentation development process
 
-What's new content is published to the website through the website CMS.
-To add a new note, browse to the [What's new CMS collection](https://admin.grafana.com/content-admin/#/collections/whats-new).
+What's new content is published to the website through the website content management system (CMS).
 
 Because this platform is meant to be used by the entire organization, by default anyone can contribute and publish to What's new, without the need for approval.
 _Quality assurance is a conversation within and between contributing teams and internal stakeholders_, but there are some best practice guidelines described in the last two sections of this topic.
 
-Enter release notes into the CMS two to four weeks before the feature is available, depending on the size of the product or feature.
-This gives the Go To Market (GTM) team time for promotion and enablement.
-For Grafana versioned releases, have your content entered in the CMS by the cut-off date communicated by the delivery team.
-For more information, refer to the [Record Announce Document Ship (RADS) guidelines](https://wiki.grafana-ops.net/w/index.php/Engineering/RADS).
+**What does "Published" mean?**
 
 It's important to understand that, in the context of the CMS, the word "published" has a slightly different meaning than in general use:
 
 - **Published**: Your entry is complete and in **Published** status.
   It's either visible on the external _What's new in Cloud_ page or is going to automatically become visible on the release date.
 - **Live**: Your entry is visible on the _What's new in Cloud_ page.
```

Comment:
So does Published and Live sometimes mean the same thing?

Reply:
No, what we're getting at is that published in this context doesn't mean live, though it does in most other contexts.

Reply:
the wording still leads me to believe that they can _sometimes_ be the same thing. "...visible on the external _What's new in Cloud_ page" and "Your entry is visible on the _What's new in Cloud_ page." are equivalent to me.

Reply:
Ah okay, I get what you mean. So I actually answered your first question incorrectly. Something can be published and not live as well as published and live. It's dependent on release date. Let me see if I can find a way to make that clearer.

Reply:
gotcha! Maybe an example would help here.